<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select</class>
 <widget class="QWidget" name="Select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="availableListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="selectedListWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>》》</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>《《</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
